### ISSUE TITLE
Request - 32-bit Windows Build

### ISSUE THREAD

**Issue by richb93** [kraxarn/spotify-qt#109]
Yeah yeah, I'm sure I'll hear something along the lines of get with the times old man etc etc.... but, I'm using a tiny PC on a stick device which runs Windows 10 32-bit, and I'd like a lightweight player to replace the slow bloaty native client.

Any chance of a 32-bit Windows build?

Please and thank you.

**Comment by kraxarn** [comment]
In that case I'm more surprised you aren't running a lightweight operating system to match :p

There's no real reason for it to be 64-bit, as it should never use that much memory to require it, it's just easier for me to set up a build pipeline for 64-bit. The automated Windows builds are still pretty temporary, and I have plans to use a proper MSVC pipeline, as soon as I get more used to how the Windows builds process works, which builds for 32-bit by default.

TL;DR: the current Windows builds are only temporary, and once I get it set up properly, Windows builds will be 32-bit.

**Comment by richb93** [comment]
Having W10 on it is quite nice because I can power said stick from a USB battery and using the Wi-Fi AP it puts out, connect from my iPad so I have a lightweight Windows client for work etc.

Thank you very much for the quick feedback - looking forward to it!

Sorry to go somewhat offtopic but I was just wondering, how easy would it be to make a Spotify Connect daemon that just runs quietly, waiting to accept connections? The reason I ask is because I run said compute stick headless and currently control it via Spotify Connect using the crappy official client. Spotiamp was lovely, but was then sadly found to be compromised from a security standpoint.... anyway apologies for not observing proper GitHub etiquette - these messages should be a bit more focused, and not someone just waffling on...

**Comment by kraxarn** [comment]
Check out [librespot](https://github.com/librespot-org/librespot) (or any of its ports), or [spotifyd](https://github.com/Spotifyd/spotifyd), sounds like what you're looking for. Similar functionality is planned for spotify-qt, but who knows how long that will take.

**Comment by kraxarn** [comment]
Looking more into this, it's worth noting that Qt 6 dropped support for 32-bit, and Qt 5 is no longer supported, so 32-bit Windows isn't a supported platform by Qt anymore. I will probably still set up 32-bit builds with Qt 5 and 64-bit builds with Qt 6, as Qt 5 is still supported by spotify-qt.
